.github/ISSUE_TEMPLATE/icon_update.yml

```
@@ -0,0 +1,67 @@
+name: Icon update
+description: Help us improve by reporting outdated icons
+title: "Update: "
+labels: [icon outdated]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `icon outdated`
+        label][icon-outdated-issues] (including closed issues) before you continue. If
+        you find one for the brand you're reporting then leave a comment on it or add
+        a reaction. For more details see the [Contributing
+        Guidelines][contributing-guidelines].
+
+
+        [contributing-guidelines]:
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+
+        [icon-outdated-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22+is%3Aopen
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. We strongly encourage you to do
+        some research here as it will help implement your request faster, although you
+        can put "None" if you are unable to find any resources.
```

This should probably be updated in line with the icon request form:

```suggestion
        Media kits, brand guidelines, SVG files, etc. You can set
        this to none if you are unable to find any resources.
```